elementary/monitor/data_monitoring/data_monitoring_alerts.py

```diff
@@ -30,17 +32,64 @@ def __init__(
         self,
         config: Config,
         tracking: AnonymousTracking,
+        filter: Optional[str] = None,
         force_update_dbt_package: bool = False,
         disable_samples: bool = False,
         send_test_message_on_success: bool = False,
     ):
-        super().__init__(config, tracking, force_update_dbt_package, disable_samples)
+        super().__init__(
+            config, tracking, force_update_dbt_package, disable_samples, filter
+        )
+        self.filter = self._parse_filter(self.raw_filter)
         self.alerts_api = AlertsAPI(
-            self.internal_dbt_runner, self.config, self.elementary_database_and_schema
+            self.internal_dbt_runner,
+            self.config,
+            self.elementary_database_and_schema,
         )
         self.sent_alert_count = 0
         self.send_test_message_on_success = send_test_message_on_success
 
+    def _parse_filter(
+        self, filter: Optional[str] = None
+    ) -> Optional[DataMonitoringAlertsFilter]:
+        if filter:
+            if self.user_dbt_runner:
+                self.tracking.set_env("select_method", "dbt selector")
+                selector_api = SelectorAPI(self.user_dbt_runner)
+                node_names = selector_api.get_selector_results(selector=filter)
+                return DataMonitoringAlertsFilter(node_names=node_names)
+
+            else:
+                data_monitoring_filter = DataMonitoringAlertsFilter()
+                tag_regex = re.compile(r"tag:.*")
+                owner_regex = re.compile(r"owner:.*")
+                model_regex = re.compile(r"model:.*")
```

Comment:
Mentioning here as well - when selecting models in dbt you don't prefix with `model:`
So it's inconsistent with the dbt project mode (and how people are used to selectors)

So I think it would also make sense to support `edr monitor -s customers` rather than `edr monitor -s model:customers`